### PR TITLE
Fixed the issue with error logs in PLUGIN-234

### PR DESCRIPTION
**Pull request by bajram-adapt**
Cherrypick for jira : https://cdap.atlassian.net/browse/PLUGIN-234

conflict from https://github.com/AdaptiveScale/google-cloud/commit/fd1a67f5ab4d687ea7f830232816f3ff61e0e3bd being in dev but not in release 0.14 resolved
Or Pr: #441

### CODE REVIEW

File: src/main/java/io/cdap/plugin/gcp/spanner/source/SpannerRecordReader.java [hunk 1]
```diff
@@ -54,7 +55,7 @@ public void initialize(InputSplit inputSplit,
       Configuration configuration = taskAttemptContext.getConfiguration();
       boolean isServiceAccountFilePath = SpannerConstants.SERVICE_ACCOUNT_TYPE_FILE_PATH
         .equals(configuration.get(SpannerConstants.SERVICE_ACCOUNT_TYPE));
-      Spanner spanner = SpannerUtil.getSpannerService(configuration.get(SpannerConstants.SERVICE_ACCOUNT),
+       spanner = SpannerUtil.getSpannerService(configuration.get(SpannerConstants.SERVICE_ACCOUNT),
```
Comment (CuriousVini) [inline src/main/java/io/cdap/plugin/gcp/spanner/source/SpannerRecordReader.java]: nit: indentation

**Review by CuriousVini — APPROVED** [review]
fix and squash

**Comment by CuriousVini** [comment]
Also please add cherrypick PR to the PR description

File: src/main/java/io/cdap/plugin/gcp/spanner/source/SpannerRecordReader.java (hunk 1 above)
Comment (bajram-adapt) [reply src/main/java/io/cdap/plugin/gcp/spanner/source/SpannerRecordReader.java]: fixed

**Pull request merged**